### PR TITLE
guides: styling to relate own-line links to the line above [ch5388]

### PR DESCRIPTION
**Pull request by maiamcc**
@hyu i didn't get to the accompaning github icon suggestion, but
this seemed like a good start!

### CODE REVIEW

**Comment by maiamcc** [comment]
Screen Shot 2020-03-25 at 6 31 48 PM

(Is the bold too much?)

File: docs/example_go.md [hunk 1]
```diff
@@ -18,13 +18,11 @@ We'll use Tilt to:
 - Measure the time from a code change to a new process
 - Optimize that time for fast feedback
 
-All the code is in this repo:
+All the code is in this repo: [tilt-example-go](https://github.com/windmilleng/tilt-example-go)
+{:.link-own-line}
```
Comment (nicks) [inline docs/example_go.md]: Putting this in one paragraph, and then using css to move it into multiple paragraphs, seems very semantically odd.

Have you considered doing this with pure CSS and not making any text changes at all?

e.g.,

```
p > a:only-child {
  text-indent: 2em;
  font-weight: bold;
}
```

Comment (maiamcc) [reply docs/example_go.md]: I considered that but the main thing I'm trying to do is get rid of the extra whitespace between the link and the paragraph above, so it's clear what bit of text the link "belongs" to; and messing with negative margins seemed foolhardy if say, the paragraph margins change from desktop to mobile. if the margins are consistent enough though, I'd happily do
```
p > a:only-child {
  text-indent: 2em;
  font-weight: bold;
  margin-top: -1em;
}
```
or whatever.

Comment (nicks) [reply docs/example_go.md]: ya, i like that suggestion. we typically set margin-bottom to $spacing-unit / 2, so maybe do margin-top: - $spacing-unit / 4

Comment (maiamcc) [reply docs/example_go.md]: cool, done.

**Pull request merged**